Update build scripts to Visual Studio 2026

rockfordroeNG opened the pull request:
The build instructions state Visual Studio 2026 is needed yet the dependency build scripts still use Visual Studio 2022.

rockfordroeNG committed:
platforms/windows-x86/external.sh: Update to VS2026

rockfordroeNG committed:
platforms/windows-x64/external.sh: Update to VS2026

toxieainc commented:
There is no need to switch yet. The building instructions just mention VS2026, because its the easiest to find/download.
The first windows build line mentions VS2019 and above.

toxieainc commented:
I also just clarified that in the line of the readme that could have been misinterpreted.

rockfordroeNG committed:
Merge branch 'vpinball:master' into master

rockfordroeNG committed:
linux: remove outdated linker flags

-Wl,--copy-dt-needed-entries breaks non-bfd linkers and is unnecessary in modern versions of bfd/gcc

rockfordroeNG committed:
Revert "linux: remove outdated linker flags"

This reverts commit b08f7fd97834b37cf1dbd3ab1faae0180ae07438.
